### ISSUE TITLE
FT don't use full CPU Resources ?

### ISSUE THREAD

**Issue by ZippelZappel** [freqtrade/freqtrade#2053]
## Step 1: Have you search for this issue before posting it?

Yes, found a similar problem description. But you wrote it is solved in an never version, i think it was in april.

## Step 2: Describe your environment
  
Python 3.6.8
CCXT 1.18.860
FT 2019.6-dev

## Step 3: Describe the problem:
On my Notebook FT produces 100% Load on all Cores when i run hyperopt.
I just installed a machine with more Compute Power to save time with higher epochs.
With sysbench i pre-checked the ability of the system to handle big workloads.
It produces 100% load on all (32) cores -> nice horizontal line at 100% as long as the calculation runs :)
FT calculates the first 32 evals very quickly with 100% load, an then it has any problems
and the load falls to near zero, then pops up all cores every x seconds, but never @100%.

![grafik](https://user-images.githubusercontent.com/52954209/61803519-943af380-ae32-11e9-9c66-7d9bf0fcd446.png)

**Comment by hroff-1902** [comment]
Yes, you are right, it's a known issue. See https://github.com/freqtrade/freqtrade/pull/1765 for details and for the solution (postponed yet).

**Comment by hroff-1902** [comment]
> Yes, found a similar problem description. But you wrote it is solved in an never version, i think it was in april.

Have missed this your note above, sorry. Was a bit in hurry answering...

Yes, I implemented the solution in April, but postponed it since I found that pickling data in the worker projects takes long time (probably, this needs more experiments to clarify the behavior, maybe anything is okay) and thus it also seems to be non-optimal.

I'm going to merge current codebase into that working branch to continue working on it.
